modules/UI/authentication/AuthHandler.js

```diff
@@ -13,12 +13,13 @@ const logger = require('jitsi-meet-logger').getLogger(__filename);
 
 let externalAuthWindow;
 let authRequiredDialog;
-
+let isForcePopupExternalAuth;
```

Comment:
Does this variable need to be scoped to the module? Seems like it can be scoped to doExternalAuth or be assigned a value here.